equinix/fabric_port_read_schema.go

```diff
@@ -308,12 +308,9 @@ func readFabricPortResourceSchema() map[string]*schema.Schema {
 			},
 		},
 		"lag": {
-			Type:        schema.TypeSet,
+			Type:        schema.TypeBool,
```

Comment:
This change should be included in the docs and included in the PR description

Reply:
Looks like it affects the datasource for fabric_port and fabric_ports.

https://registry.terraform.io/providers/equinix/equinix/latest/docs/data-sources/equinix_fabric_port#nested-schema-for-lag

Reply:
Added link to API documentation and more information about this change to the PR description.

Reply:
I think it would be preferable to add a new `lag_enabled` property, which lines up with the new `lagEnabled` property in the API spec.  Has support for the old `lag` property been dropped entirely?  If a customer has an existing terraform configuration that uses
```hcl
lag {
  enabled = true
  name = "some name"
  # ...
}
```
and they run `terraform plan` / `terraform apply` right now, using the current release of the terraform provider, will they get an error or does is still work?

Reply:
Yes, the API spec has been updated from the older implementation and the customers might need to change their terraform scripts to accommodate the changed fields. 
cc @anchala71

Reply:
There is no Customer for the current version of Port specifications and we did release for a beta versions. I think it would be safe to deprecate the field. Anyways let me add team who involved in this change @askamanth-equinix

Reply:
@askamanth-equinix

Reply:
@ctreatma Ack. I have updated the property to lag_enabled.

Reply:
Since no customer is using this feature as of now, we can go ahead and release this.

Reply:
@ctreatma I have updated this field now.